cxbox-api/src/main/java/org/cxbox/api/data/dto/rowmeta/FieldDTO.java

```diff
@@ -204,11 +206,20 @@ public void clearFilterValues() {
 		filterValues.clear();
 	}
 
+	/**
+	 * @deprecated Since 4.0.0-M10
+	 * use {@link FieldDTO#setIconAllValue}
+	 */
+	@Deprecated(since = "4.0.0-M10")
 	public void setIconWithValue(String val, IconCode icon, boolean isFilterValue) {
```

Comment:
@deprecated method is ready to be deleted

Reply:
done